Makefile

```
@@ -6,7 +6,7 @@
 .PHONY: help build build-dev build-prod run run-dev run-prod stop clean test lint format check install
 .PHONY: build-info clean-cache build-no-cache inspect analyze health validate perf setup deploy
 .PHONY: dev prod prod-nginx logs-dev logs-prod shell-dev shell-prod ps restart-dev restart-prod
-.PHONY: update push security monitor test-docker fix
+.PHONY: update push security monitor test-docker fix build-index list-components
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)